specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/examples/BillingActivityList.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-privatepreview",
+    "billingAccountName": "7303440",
+    "startDate": "{2019-01-10}",
+    "endDate": "{2020-02-10}",
+    "activityLogEventCategory": "ManageAccountAdd"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList",
+            "name": "99a43ce1-443c-40e4-a9f0-d23152ee971b",
+            "properties": {
+              "activityEventId": "ManageAccountAdd",
+              "billingAccountName": "7303440",
+              "createdOn": "2020-01-10 19:44:00.5300000",
+              "createdByPrincipalUserEmailAddress": "demouser@outlook.com",
+              "friendlyEventCategoryName": "Add Account",
+              "extendedProperties": "{\"AccountEmail\":\"demouser@outlook.com\",\"AccountAuthType\":\"OrganizationalAccount\",\"AccountName\":\"demouseraccount@miccrosoft.com\",\"AccountCostCenter\":\"ACM\"}"
+            },
+            "type": "Microsoft.Billing/billingAccounts/billingActivity"
+          },
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList",
```

Comment:
the id must end in the 'name' - so the id should be /providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList/cd1c3b05-e2e5-4671-af97-608eb65c9991 .. and similarly for above entry